.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 6.5.0
```

Comment:
Noting the version of node that works for ASDF. Let me know if you'd prefer to not have the clutter of the extra file